Makefile

```diff
@@ -25,7 +25,7 @@ endif
 
 VENDORED ?= 0
 ifeq ($(VENDORED),1)
-	ARGS += "--frozen"
+	ARGS += "--frozen" "--offline"
```

Comment:
Only --offline should be required

Reply:
That said, it is probably more safe to just pass both

Reply:
Interestingly, `--offline` doesn't seem to imply `--frozen`. In some cases, `--offline` builds can work without a `Cargo.lock` file already existing (when there's a local copy of the registry and dependent crates, and there are no git dependencies).

So perhaps it makes sense to pass both.